### PR TITLE
CSS overflow issue on demo

### LINKED ISSUE #333: CSS overflow issue on demo
See: timble/kodekit-ui#59

### PR DESCRIPTION
**Pull request by ercanozkaya**
Closes #333

### CODE REVIEW

**Comment by ercanozkaya** [comment]
@yiendos Could you give this a go please if you have an iOS device?

File: code/libraries/joomlatools/library/resources/assets/css/build/admin.css [hunk 1]
```diff
@@ -3277,7 +3277,6 @@ body.k-is-resizing {
 }
```
Comment (yiendos) [inline code/libraries/joomlatools/library/resources/assets/css/build/admin.css]: @ercanozkaya I'm a little bit confused: 
https://demo.joomlatools.box/media/koowa/com_koowa/css/admin.css?04e0c297

This is where the styling for the following page comes from:
https://demo.joomlatools.box/en/component/textman/?view=article&layout=form&menu_id=371&id=20

Looking at how the joomlatools-framework is symlinked on an example site: 

```
[10:30 AM]-[vagrant@joomlatools]-[/var/www/debug/media/koowa] 
$ ls -la
total 0
drwxr-xr-x  4 501 dialout  128 Dec  2  2019 ./
drwxrwxr-x 32 501 dialout 1024 Dec  2  2019 ../
lrwxrwxrwx  1 501 dialout  104 Dec  2  2019 com_koowa -> /home/vagrant/Projects/joomlatools-framework/code/libraries/joomlatools/component/koowa/resources/assets/
lrwxrwxrwx  1 501 dialout   96 Dec  2  2019 framework -> /home/vagrant/Projects/joomlatools-framework/code/libraries/joomlatools/library/resources/assets/
``` 

So therefore: 

https://demo.joomlatools.box/media/koowa/com_koowa/css/admin.css?04e0c297

Relates to: 
https://github.com/joomlatools/joomlatools-framework/tree/master/code/libraries/joomlatools/component/koowa/resources/assets/css/admin.css 

But we haven't made changes to this stylesheets, rather the PR relates to: 

* `code/libraries/joomlatools/library/resources/assets/css/admin.css`
* `code/libraries/joomlatools/library/resources/assets/css/build/admin.css`

**Comment by yiendos** [comment]
@ercanozkaya if I replace the demo site admin.css with either the build/admin.css or minified admin.css this fix works great on iphone 6 via browserstack.  I'll just quickly run over some further tests on different browsers and OS after lunch.

**Comment by yiendos** [comment]
## Browserstack results for: 
https://demo.joomlatools.box/en/component/textman/?view=article&layout=form&menu_id=371&id=20

Seems to have a good coverage for the latest type of phones 

**iphone 6**
	safari	good 
	chrome	good 

**iphone 7** 
	safari	good 
	chrome 	good 

**iphone 8** 
	safari 	good 
	chrome 	good

**iphone 11 pro**
	safari	good 
	chrome	good

**Galaxy S9** 
	chrome	good 

**Galaxy S8** 
	chrome 	good 

**Pixel 3A XL** 
	chrome 	good 

**Pixel 3A** 
	chrome 	good	

**Pixel 2**
	chrome	good

**Pixel XL** 
	chrome	good

**Comment by ercanozkaya** [comment]
@yiendos You are totally right. Did a recompile for the other admin.css file. Can you do a quick test please? No need to test every device again, just a single confirm is enough.

**Comment by yiendos** [comment]
@ercanozkaya confirmed working like a treat. Follow on question, how come we have two css locations:
code/libraries/joomlatools/component/koowa/resources/assets/css/admin.css
vs 
code/libraries/joomlatools/library/resources/assets/css/admin.css

Does the former actually ever get used in the UX at all?

**Comment by ercanozkaya** [comment]
@yiendos Not right now, no. I think this is something I need to streamline as well. There are too many CSS files around

**Comment by johanjanssens** [comment]
@ercanozkaya Is this something that is ready?

**Comment by yiendos** [comment]
@johanjanssens all good from my side

**Comment by johanjanssens** [comment]
@ercanozkaya @yiendos Can you guys let me know once this is deployed so i can inform the user who reported the issue? Thanks!

**Comment by johanjanssens** [comment]
@johanjanssens Note to self, merged in master will go out with next releases

**Pull request merged**